src/kvstore/RocksEngineConfig.cpp

```diff
@@ -171,7 +171,7 @@ rocksdb::Status initRocksdbOptions(rocksdb::Options &baseOpts) {
         bbtOpts.no_block_cache = true;
     } else {
         static std::shared_ptr<rocksdb::Cache> blockCache
-            = rocksdb::NewLRUCache(FLAGS_rocksdb_block_cache * 1024 * 1024, 8/*shard bits*/);
+            = rocksdb::NewLRUCache(FLAGS_rocksdb_block_cache * 1024 * 1024, 12/*shard bits*/);
```

Comment:
why modify the default value ?

Reply:
2^8 = 256 shards seem a little bit small for modern machines with nearly 100 CPUs.

Reply:
keep unchanged.